.github/workflows/preview-cleanup.yml

```
@@ -3,6 +3,8 @@ name: PR preview cleanup
 on:
   pull_request:
     types: [closed]
+    branches-ignore:
+      - master
```

Would it make sense to just whitelist this for develop?
```suggestion
    branches:
      - develop
```

---

Makes sense